src/main/webapp/view/lecture-view.css

```diff
@@ -1,5 +1,18 @@
 /* Layout */
 
+body,
+html {
+  height: 100vh;
+  max-height: 100vh;
+  max-width: 100vw;
+  width: 100vw;
+  overflow: hidden;
```

Comment:
don't forget to alphabetize the declarations :)

Reply:
i knew i'd miss one! thanks for catching it